spec/lib/sanitizer_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+describe Sanitizer, ".sanitize" do
+  subject(:sanitize) { Sanitizer.sanitize(string) }
```

Comment:
I'm not a huge fan of RSpec's subject feature, so this may reflect my ignorance. Why do you use `expect(sanitize).to` in some specs, and `is_expected.to` in others?

Reply:
I use the `is_expected` syntax for simple one-liners like:
```ruby
it { is_expected.to eq '' }
```
That's the equivalent of:
```ruby
it 'is nil' do
  expect(sanitize).to be_nil
end
```
I use the former in cases where no further explanation of the behavior seems required, and the latter much more frequently in order to clarify the behavior. I'll often use the one-liner syntax for a base case spec, and the wordier syntax for the more detailed specs. (I don't have a particular rule for using one or the other beyond a gut check of when I feel like some extra verbiage would help.)

We also use the [Shoulda](http://matchers.shoulda.io/) gem for similar one-line matchers for models, controllers, etc. Those generally require a `subject`.